frontend/packages/kubevirt-plugin/src/constants/vm/storage/index.ts

```diff
@@ -1,4 +1,6 @@
+export * from './access-mode';
 export * from './data-volume-source-type';
 export * from './disk-bus';
 export * from './disk-type';
 export * from './volume-type';
+export * from './volume-mode';
```

Comment:
alphabetically `volume-mode` is before `volume-type` :-)

Reply:
You are correct! I'll fix it in a follow-up. I want to fix a few misspelled function names that I found, but didn't want to do it in this PR. I'll fix it with those.